### PR TITLE
Release 26.0.2snap1

### PR DESCRIPTION
**Pull request by kyrofa**
- import: add documentation about confinement
- php: add config flag --enable-sysvsem
- nextcloud: update to 26.0.2

### CODE REVIEW

**Review by Pilzinsel64 — APPROVED** [review]
Beeing on `latest/beta` for a couple days now. Everything seems to work fine on my prod.

Sadly I hadn't the time to test a fresh install as I need/want to setup a VM environment for at first. Maybe tomorrow. But for now, approve this as maybe someone else want to test it in the meantime. :D

**Comment by scubamuc** [comment]
Testing `--channel=latest/beta`

```
-------------------------------------------------
       Snap Version auf dem System!              

	-- Aktuelle Snap Version: 
snap    2.59.4
snapd   2.59.4
series  16
ubuntu  22.04
kernel  5.15.0-71-generic

	-- Aktuelle Nextcloud Snap: 
Name       Version                    Rev    Tracking     Publisher   Notes
nextcloud  25.0.7snap1+git11.0c1f571  35638  latest/beta  nextcloud✓  -

	-- Nextcloud Snap Dienste:
				
Service                    Startup  Current   Notes
nextcloud.apache           enabled  active    -
nextcloud.logrotate        enabled  inactive  timer-activated
nextcloud.mdns-publisher   enabled  active    -
nextcloud.mysql            enabled  active    -
nextcloud.nextcloud-cron   enabled  active    -
nextcloud.nextcloud-fixer  enabled  inactive  -
nextcloud.php-fpm          enabled  active    -
nextcloud.redis-server     enabled  active    -
nextcloud.renew-certs      enabled  active    -

       ================================== 
```
no issues with fresh install

Verion numbering seems off:
grafik

**Comment by pachulo** [comment]
Upgraded my instance and a few tests show that everything looks and works nice!

> Verion numbering seems off:

Yes, that's expected @scubamuc , till this PR is merged into the `master` branch.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Nextcloud server packaged as a snap. It consists of:
 
-- Nextcloud 25
+- Nextcloud 26
 - Apache 2.4
 - PHP 8.0
```
Comment (pachulo) [inline README.md:7]:
```suggestion
- PHP 8.1
```

**Review by pachulo — APPROVED** [review]

**Comment by kyrofa** [comment]
Hmm... the default from-scratch UX is kind of garbage here. Once you create an admin account, it immediately jumps into maintenance mode and stays there until you refresh.

**Comment by Pilzinsel64** [comment]
@kyrofa This did happen frequently on the past to me. I am on a RPi4, maybe my system is too slow. If it took too long I just restarted the snap and everything went fine.
For NC26 I hadn't this problem yet, if I am remember correctly.

**Comment by kyrofa** [comment]
Oh so this isn't a new issue? Okay, good deal, I'll go ahead and merge then. We should fix that, though.

**Comment by scubamuc** [comment]
No, I Had No issues.

**Pull request merged**